src/js/helpers/cleanData.js

```diff
@@ -89,8 +89,10 @@ export const cleanContact = contact => {
 };
 
 export const cleanLocationPageJanisUrl = janisUrl => {
-  // quick fix for urls until we get localized urls working in joplin
-  return `/${janisUrl.split('/en/')[1]}`;
+  if(!!janisUrl.length){
+    return janisUrl[0];
```

Comment:
why is janisUrl an array here? not something that needs to be changed rn but might be nice to have a comment about

Reply:
good question! the janisUrls that come back are the array of urls for the page, so its dept and topic pages instances, whereas before I believe it was just one url.